test/unit/app/controllers/network/network-controller-test.js

```diff
@@ -34,8 +34,10 @@ describe('NetworkController', function () {
         assert.equal(providerProxy.test, true)
       })
     })
+
     describe('#getNetworkState', function () {
       it('should return loading when new', function () {
+        networkController = new NetworkController()
```

Comment:
Nit: Instead of constructing this again, maybe the `initializeProvider` call could be moved out of the `beforeEach` and into each test that needs that step executed? That'd avoid the redundant `networkController` construction in this test.

Reply:
Very fair, I was lazy. Fixed in: b3e5bef